tasks-by-users/Rehan-Khan-Engineer/Bikes-webpage/Rehan.css

```diff
@@ -0,0 +1,42 @@
+/* Bike ki Dukaan */
+body{ margin: 0; font-family: Arial; background: white;}
+
+#wrapper {margin: 0 auto; width: 980px; background: white;}
```

Comment:
different property on different line please @nehalskdev

Reply:
@coder-mezab , i think its the PR of @Rehan-Khan-Engineer  :D

Reply:
> different property on different line please @nehalskdev

@Rehan-Khan-Engineer , install prettier and enable format on save setting. it will be fixed !

Reply:
@faran4engg  huh ofc pardon me @nehalskdev

Reply:
Done @coder-mezab